pkg/controller/servicebindingrequest/retriever.go

```diff
@@ -28,6 +28,24 @@ const (
 	secretPrefix  = "urn:alm:descriptor:servicebindingrequest:env:object:secret"
 )
 
+func getNestedValue(key string, sectionMap interface{}) (string, error) {
+	var value interface{}
+	var exists bool
+	if !strings.Contains(key, ".") {
+		value, exists = sectionMap.(map[string]interface{})[key]
```

Comment:
You can just use inline declaration here (`:=`), since after this `if` you declare them again (line 42).

Reply:
Done. 7351fde1f36e5fe43f8847745714b967842bd9dd